src/test/kotlin/uk/gov/dluhc/printapi/testsupport/assertj/assertions/TemporaryCertificateAssert.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.dluhc.printapi.testsupport.assertj.assertions
+
+import org.assertj.core.api.AbstractObjectAssert
+import org.assertj.core.util.CheckReturnValue
+import uk.gov.dluhc.printapi.database.entity.TemporaryCertificate
+import java.time.LocalDate
+import java.util.Objects
+import javax.annotation.Generated
+
+/**
+ * [TemporaryCertificate] specific assertions - Generated by CustomAssertionGenerator.
+ */
+@Generated(value = ["assertj-assertions-generator"])
```

Comment:
Just for my own underanding why is `@Generated` here?

Reply:
Ah yes, copy & paste again - this time from `CertificateAssert`. I asked @Neil-Massey and he said `CertificateAssert` was generated automatically for him. This is magic I didn't know about! I'll remove the annotation here, but it would be good to find out about this voodoo.

Reply:
Interesting! This https://plugins.jetbrains.com/plugin/10345-assertions2assertj or https://joel-costigliola.github.io/assertj/assertj-assertions-generator.html

Reply:
Nice 👍🏼